docs/getting-started/rulesets.md

```diff
@@ -282,3 +282,66 @@ except:
 ## Creating custom functions
 
 Learn more about [custom functions](../guides/custom-functions.md).
+
+## Distributing
+
+You are very welcome to share your rulesets with the world. To help you out distribute your rulesets among the others, Spectral provides a few way to load rulesets.
```

Comment:
@P0lip For the packaging standpoint, what's the recommended way to release a breaking change?

Could the `extends` ruleset property support some kind of way to describe the wanted version? It looks like `unpkg.com` supports loading with a semver specifier.

Reply:
> Could the extends ruleset property support some kind of way to describe the wanted version? It looks like unpkg.com supports loading with a semver specifier.

Yes, we support versioning, so you can peg the package at a certain version.

Should I mention it?

Reply:
@P0lip Yes, please! Could you please tweak your example so that it also demonstrates how to reference a npm ruseset by its major version for instance?

Reply:
Sounds good to me. On it.